src/App.tsx

```diff
@@ -1,42 +1,176 @@
-import React, { useEffect } from "react";
-import { Provider as PaperProvider, MD3DarkTheme } from "react-native-paper";
-import { DarkTheme, NavigationContainer } from "@react-navigation/native";
-import { AppNavigator } from "./navigation/AppNavigator";
-import { DwnService } from "./features/dwn/dwn-service";
-import { enableLegendStateReact } from "@legendapp/state/react";
-import { StatusBar } from "expo-status-bar";
-import { linking } from "./navigation/deep-links";
-import { bootstrapIdentityAgent } from "./features/identity/identity-agent";
-
-enableLegendStateReact();
-
-export const theme: typeof MD3DarkTheme = {
-  ...MD3DarkTheme,
-  colors: {
-    ...MD3DarkTheme.colors,
-    primary: "#ffec19",
-  },
-};
+import React from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import {
+  type BottomTabNavigationOptions,
+  createBottomTabNavigator,
+} from "@react-navigation/bottom-tabs";
+import { userProfiles } from "@/services/profile.service";
+import Octicons from "@expo/vector-icons/Octicons";
+import WelcomeScreen from "@/pages/onboarding/welcome/Welcome";
+import ConnectionDetailScreen from "@/pages/default/connections/ConnectionDetail";
+import ReviewConnectionScreen from "@/pages/default/connections/ReviewConnection";
+import AddCredentialsScreen from "@/pages/default/credentials/AddCredentials";
+import CredentialDetailScreen from "@/pages/default/credentials/CredentialDetail";
+import DiscoverScreen from "@/pages/default/discover/Discover";
+import ProfileDetailScreen from "@/pages/default/profiles/ProfileDetail";
+import AuthScreen from "@/pages/onboarding/auth/Auth";
+import CreateScreen from "@/pages/onboarding/create/Create";
+import ImportScreen from "@/pages/onboarding/import/Import";
+import ProfilesScreen from "@/pages/default/profiles/Profiles";
+import CredentialsScreen from "@/pages/default/credentials/Credentials";
+import ConnectionsScreen from "@/pages/default/connections/Connections";
+import ActivityScreen from "@/pages/default/activity/Activity";
+import { type LinkingOptions } from "@react-navigation/native";
+import { ColorTheme, DefaultTheme } from "@/theme/colors";
+import { Space } from "@/theme/layouts";
+import { Typography } from "@/theme/typography";
+import AddProfileScreen from "@/pages/default/profiles/AddProfile";
+import AddCredentialDetailScreen from "./pages/default/credentials/AddCredentialDetail";
+import AddCredentialOptionsScreen from "./pages/default/credentials/AddCredentialOptions";
 
-export default function App() {
-  useEffect(() => {
-    const startupTasks = async () => {
-      await DwnService.initExpoLevelDwn();
-      await bootstrapIdentityAgent(
-        "passphrase",
-        "Personal",
-        DwnService.getDwn()
-      );
-    };
-    void startupTasks();
-  }, []);
+const App = () => {
+  const Stack = createNativeStackNavigator();
```

Comment:
- you don't want this inside of your function body like this. needs to be moved outside of render